dtool/Package.cmake

```diff
@@ -0,0 +1,703 @@
+set(THIRDPARTY_DIRECTORY "" CACHE PATH
+  "Optional location of a makepanda-style thirdparty directory. All libraries
+   located here will be prioritized over system libraries. Useful for
+   cross-compiling.")
+
+if(THIRDPARTY_DIRECTORY)
+  # This policy is necessary for PackageName_ROOT variables to be respected
+  if(POLICY CMP0074)
+    cmake_policy(GET CMP0074 _policy_cmp0074)
+  endif()
+
+  if(NOT _policy_cmp0074 STREQUAL "NEW")
+    message(FATAL_ERROR
+      "Your version of CMake is too old; please upgrade or unset THIRDPARTY_DIRCTORY to continue.")
+  endif()
+
+  # Dig up the actual "libs" directory
+  if(APPLE)
+    set(_package_dir "${THIRDPARTY_DIRECTORY}/darwin-libs-a")
+
+  elseif(WIN32)
+    if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+      set(_package_dir "${THIRDPARTY_DIRECTORY}/win-libs-vc14-x64")
```

Comment:
Probably instead of assuming `vc14` we should check the MSVC version (though, I note that 2015, 2017, and 2019 are all minor versions under "14", bless Microsoft, I don't know whether this is future-proof enough)

Reply:
I figure after makepanda dies, the thirdparty directory should be structured in a way that's more CMake-specific (unless thirdparty zips go down with the ship). This bit is just meant to help people hit the ground running in the transition to CMake; it'll definitely evolve in the near future.

Reply:
Fine with me